interfaces/interfaces_helpers.go

```diff
@@ -0,0 +1,82 @@
+package interfaces
+
+import (
+	"bufio"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/TudorHulban/GoLayouter/objects"
+)
+
+const _pathInput = "../test_cases/folder_c6"
+
+// to evoid imported cycle
+func IRWritterReadFile(filePath string) ([]string, error) {
+	fileHandler, errOp := os.Open(filePath)
+	if errOp != nil {
+		return nil, errOp
+	}
+
+	var errClo error
+	defer func() {
+		errClo = fileHandler.Close()
+	}()
+
+	var res []string
+
+	scanner := bufio.NewScanner(fileHandler)
+	for scanner.Scan() {
+		res = append(res, scanner.Text())
+	}
+
+	return res, errClo
+}
+
+func TestConvertToIWritter(t *testing.T) {
+	content, errRead := IRWritterReadFile(_pathInput)
+	require.NoError(t, errRead)
+
+	e := objects.NewEntries(content)
+	entries := e.Parse()
+	writter := objects.ConvertToIWritter(entries)
+
+	for _, element := range writter {
+		log.Print(element)
+	}
+}
+
+func WriteToDisk(paths []IWritter) error {
```

Comment:
create an app folder with three subfolders:  
a. apperrors with doc.go in it
b. configs with a doc.go in it
c. service with a file with these helpers which maybe we  could organize in a service object

Reply:
addressed in fa38b125828f208631c482c74923ffb3fbe4f1da